### ISSUE TITLE
get_client('chem') not working

### ISSUE THREAD

**Issue by newgene** [biothings/biothings_client.py#4]
```
In [41]: biothings_client.get_client('chem')
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
<ipython-input-41-b2b228d0f801> in <module>()
----> 1 biothings_client.get_client('chem')

~/opt/devpy3/lib/python3.6/site-packages/biothings_client/__init__.py in get_client(biothing_type, instance, *args, **kwargs)
    164     biothing_type = biothing_type.lower()
    165     if (biothing_type not in CLIENT_SETTINGS and not kwargs.get('url', False)):
--> 166         raise Exception("No client named '{0}', currently available clients are: {1}".format(biothing_type, list(CLIENT_SETTINGS.keys())))
    167     _settings = CLIENT_SETTINGS[biothing_type] if biothing_type in CLIENT_SETTINGS else _generate_settings(biothing_type, kwargs.get('url'))
    168     _class = type(_settings["class_name"], tuple([_settings["base_class"]] + _settings["mixins"]),

Exception: No client named 'chem', currently available clients are: ['gene', 'variant', 'taxon', 'drug']
```

even though "chem" is registered here: https://github.com/biothings/biothings_client.py/blob/master/biothings_client/__init__.py#L145

**Comment by cyrus0824** [comment]
This is probably a version issue...when I install from the head of the github repo using:

```
pip install git+https://github.com/biothings/biothings_client.py
```

it works fine:

```
In [1]: from biothings_client import get_client

In [2]: mc = get_client("chem")

In [3]: mc.metadata()
Out[3]:
{'build_date': '2018-09-14T20:08:20.749610',
 'build_version': '20180914',
 'src': {'aeolus': {'license': 'CC0 1.0',
   'license_url': 'http://datadryad.org/resource/doi:10.5061/dryad.8q0s4',
   'license_url_short': 'https://goo.gl/pLRNT8',
   'stats': {'aeolus': 3044},
   'url': 'http://www.nature.com/articles/sdata201626',
   'version': '20160422'},
  'chebi': {'license_url': 'https://www.ebi.ac.uk/about/terms-of-use',
   'license_url_short': 'https://goo.gl/FJpLMf',
   'stats': {'chebi': 98229},
   'url': 'https://www.ebi.ac.uk/chebi/',
   'version': 'rel167'},
  'chembl': {'license_url': 'https://www.ebi.ac.uk/about/terms-of-use',
   'license_url_short': 'https://goo.gl/FJpLMf',
   'stats': {'chembl': 1828820},
   'url': 'https://www.ebi.ac.uk/chembl/',
   'version': 'ChEMBL_24'},
  'drugbank': {'license': 'CC BY-NC 4.0',
   'license_url': 'https://www.drugbank.ca/releases/latest',
   'license_url_short': 'https://goo.gl/kvVASD',
   'stats': {'drugbank': 11290},
   'url': 'http://www.drugbank.ca',
   'version': '5.1.1'},
  'drugcentral': {'license': 'CC BY-SA 4.0',
   'license_url': 'http://drugcentral.org/privacy',
   'license_url_short': 'https://goo.gl/QDNyNe',
   'stats': {'drugcentral': 4714},
   'url': 'http://drugcentral.org/',
   'version': '201707'},
  'ginas': {'license_url': '?',
   'stats': {'ginas': 58468},
   'url': 'https://ginas.ncats.nih.gov',
   'version': 'v1'},
  'ndc': {'license_url': '?',
   'stats': {'ndc': 30240},
   'url': 'http://www.fda.gov/Drugs/InformationOnDrugs/ucm142438.htm',
   'version': '2018-09-07'},
  'pharmgkb': {'license': 'CC BY-SA 4.0',
   'license_url': 'https://www.pharmgkb.org/page/dataUsagePolicy',
   'license_url_short': 'https://goo.gl/6ZW4iX',
   'stats': {'pharmgkb': 3254},
   'url': 'https://www.pharmgkb.org/',
   'version': '2018-09-05'},
  'pubchem': {'license_url': 'https://www.ncbi.nlm.nih.gov/home/about/policies/',
   'license_url_short': 'https://goo.gl/Ztr5rl',
   'stats': {'pubchem': 96225491},
   'url': 'https://pubchem.ncbi.nlm.nih.gov/',
   'version': '2018-09-01'},
  'sider': {'license': 'CC BY-NC-SA 3.0',
   'license_url': 'ftp://xi.embl.de/SIDER/LICENSE',
   'license_url_short': 'https://goo.gl/8b7ZCQ',
   'stats': {'sider': 1507},
   'url': 'http://sideeffects.embl.de/',
   'version': '2015-10-21'},
  'unii': {'license_url': '?',
   'stats': {'unii': 101775},
   'url': 'https://fdasis.nlm.nih.gov/srs/',
   'version': '2018-08-31'}},
 'src_version': {'aeolus': '20160422',
  'chebi': 'rel167',
  'chembl': 'ChEMBL_24',
  'drugbank': '5.1.1',
  'drugcentral': '201707',
  'ginas': 'v1',
  'ndc': '2018-09-07',
  'pharmgkb': '2018-09-05',
  'pubchem': '2018-09-01',
  'sider': '2015-10-21',
  'unii': '2018-08-31'},
 'stats': {}}```

Even if I install from pypi, using:

```
pip install biothings_client
```

it seems to work fine:

```
In [1]: from biothings_client import get_client

In [2]: mc = get_client("chem")

In [3]: mc.metadata()
Out[3]:
{'build_date': '2018-09-14T20:08:20.749610',
 'build_version': '20180914',
 'src': {'aeolus': {'license': 'CC0 1.0',
   'license_url': 'http://datadryad.org/resource/doi:10.5061/dryad.8q0s4',
   'license_url_short': 'https://goo.gl/pLRNT8',
   'stats': {'aeolus': 3044},
   'url': 'http://www.nature.com/articles/sdata201626',
   'version': '20160422'},
  'chebi': {'license_url': 'https://www.ebi.ac.uk/about/terms-of-use',
   'license_url_short': 'https://goo.gl/FJpLMf',
   'stats': {'chebi': 98229},
   'url': 'https://www.ebi.ac.uk/chebi/',
   'version': 'rel167'},
  'chembl': {'license_url': 'https://www.ebi.ac.uk/about/terms-of-use',
   'license_url_short': 'https://goo.gl/FJpLMf',
   'stats': {'chembl': 1828820},
   'url': 'https://www.ebi.ac.uk/chembl/',
   'version': 'ChEMBL_24'},
  'drugbank': {'license': 'CC BY-NC 4.0',
   'license_url': 'https://www.drugbank.ca/releases/latest',
   'license_url_short': 'https://goo.gl/kvVASD',
   'stats': {'drugbank': 11290},
   'url': 'http://www.drugbank.ca',
   'version': '5.1.1'},
  'drugcentral': {'license': 'CC BY-SA 4.0',
   'license_url': 'http://drugcentral.org/privacy',
   'license_url_short': 'https://goo.gl/QDNyNe',
   'stats': {'drugcentral': 4714},
   'url': 'http://drugcentral.org/',
   'version': '201707'},
  'ginas': {'license_url': '?',
   'stats': {'ginas': 58468},
   'url': 'https://ginas.ncats.nih.gov',
   'version': 'v1'},
  'ndc': {'license_url': '?',
   'stats': {'ndc': 30240},
   'url': 'http://www.fda.gov/Drugs/InformationOnDrugs/ucm142438.htm',
   'version': '2018-09-07'},
  'pharmgkb': {'license': 'CC BY-SA 4.0',
   'license_url': 'https://www.pharmgkb.org/page/dataUsagePolicy',
   'license_url_short': 'https://goo.gl/6ZW4iX',
   'stats': {'pharmgkb': 3254},
   'url': 'https://www.pharmgkb.org/',
   'version': '2018-09-05'},
  'pubchem': {'license_url': 'https://www.ncbi.nlm.nih.gov/home/about/policies/',
   'license_url_short': 'https://goo.gl/Ztr5rl',
   'stats': {'pubchem': 96225491},
   'url': 'https://pubchem.ncbi.nlm.nih.gov/',
   'version': '2018-09-01'},
  'sider': {'license': 'CC BY-NC-SA 3.0',
   'license_url': 'ftp://xi.embl.de/SIDER/LICENSE',
   'license_url_short': 'https://goo.gl/8b7ZCQ',
   'stats': {'sider': 1507},
   'url': 'http://sideeffects.embl.de/',
   'version': '2015-10-21'},
  'unii': {'license_url': '?',
   'stats': {'unii': 101775},
   'url': 'https://fdasis.nlm.nih.gov/srs/',
   'version': '2018-08-31'}},
 'src_version': {'aeolus': '20160422',
  'chebi': 'rel167',
  'chembl': 'ChEMBL_24',
  'drugbank': '5.1.1',
  'drugcentral': '201707',
  'ginas': 'v1',
  'ndc': '2018-09-07',
  'pharmgkb': '2018-09-05',
  'pubchem': '2018-09-01',
  'sider': '2015-10-21',
  'unii': '2018-08-31'},
 'stats': {}}
```

I suspect you have a quite old version of the client, I tried both pypi versions (0.1.0 and 0.1.1) and both support "chem"...

Maybe try uninstalling and reinstalling a more current version?

**Comment by newgene** [comment]
You are right, it is the version issue. But the version I install is the latest 0.1.1. And that particular version does not have "chem" defined:

https://github.com/biothings/biothings_client.py/blob/v0.1.1/biothings_client/__init__.py

So we should make a new release then?

**Comment by newgene** [comment]
Fixed in [biothings_client v0.2.0 release](https://pypi.org/project/biothings-client/).
